client/src/components/Students.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import {
+    Alert,
+    Button,
+    Card,
+    CardBody,
+    CardHeader,
+    CardSubtitle,
+    CardTitle,
+    Col,
+    Container,
+    Modal,
+    ModalBody,
+    ModalFooter,
+    ModalHeader,
+    Row,
+    Table
+} from 'reactstrap';
+import { Redirect, Link } from 'react-router-dom';
+import ApiRequestHandler from '../entities/ApiRequestHelper';
+
+export default function Students({ useAuthHandler }) {
+    const authHandler = useAuthHandler();
+
+    const [students, setStudents] = useState([]);
+
+
+    const renderStudents = () => {
+        console.log(students)
+        return (
+            <Card>
+                <CardHeader>
+                    <CardTitle tag="h3">Student list</CardTitle>
+                </CardHeader>
+                <CardBody>
+                    <Table responsive striped hover>
+                        <thead>
+                            <tr>
+                                <th>#</th>
+                                <th>Username</th>
+                                <th>Surname</th>
+                                <th>Name</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {
+                                students && students.map((j, i) =>
+                                    <tr key={j.id} tag={Link} to={"/users/" + j.username}>
```

Comment:
Does this work? Or does it keep the table looking like a table?

Reply:
What do you mean? It show the table as a table because I wanted a table. The link does not work, I'm not sure yet why not or how can I make it work

Reply:
fix this as we talked, rebase and we're good to go